recipes/folly/all/conanfile.py

```diff
@@ -83,6 +83,13 @@ def _strip_options_requirements(self):
     def _required_boost_components(self):
         return ["context", "filesystem", "program_options", "regex", "system", "thread"]
 
+    # TODO: Remove CMake from build_requirements when folly/2019.10.21.00 is removed from CCI
```

Comment:
As @prince-chrismc said, we do not removed versions

Reply:
I don't understand what exactly  @prince-chrismc  means, other folly versions are not supported on windows, but you should think about it at least for the libraries that are released every month.
https://github.com/facebook/folly/releases